jobs/demo-district-update-pass.groovy

```diff
@@ -0,0 +1,85 @@
+// Automate monthly change for PL sandbox account passwords
+// https://khanacademy.atlassian.net/browse/DIST-4092
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+// Vars we use, under jenkins-jobs/vars/.  This is just for documentation.
+//import vars.exec
+//import vars.notify
+//import vars.withTimeout
+//import vars.withSecrets
+
+new Setup(steps
+
+).allowConcurrentBuilds(
+
+).addStringParam(
+   "CYPRESS_GIT_REVISION",
+   """The name of a cypress branch to use when building.""",
+   "update-psw"
+
+).addStringParam(
+   "SLACK_CHANNEL",
+   "The slack channel to which to send failure alerts.",
+   "#demo-district-logs"
+
+).addStringParam(
+   "REVISION_DESCRIPTION",
+   """Set by the buildmaster to give a more human-readable description
+of the GIT_REVISION, especially if it is a commit rather than a branch.
+Defaults to GIT_REVISION.""",
```

Comment:
Looks like this comment isn't true.  Should it have a default value (on line 32) at all?